bukkit/src/main/java/com/viaversion/viaversion/bukkit/providers/BukkitPickItemProvider.java

```diff
@@ -187,14 +189,15 @@ static BlockToItem build() {
                     }
                     return item;
                 };
-            } else if (PaperViaInjector.hasMethod(Material.class, "isItem")) {
-                return (block, includeData) -> new ItemStack(block.getType(), 1);
-            } else {
+            } else if (Via.getAPI().getServerVersion().lowestSupportedProtocolVersion().equalTo(ProtocolVersion.v1_8)) {
```

Comment:
It works for everything until 1.13

Reply:
The legacy data loader? No - the current implementation only works for 1.8

Reply:
That's not correct :p

Reply:
Actually I'm an idiot sorry about that